backend-api/src/functions/tests/pact/createApp.ts

```diff
@@ -12,7 +14,18 @@ export async function createApp(): Promise<Application> {
     const result = await asyncTokenHandlerConstructor({
       headers: req.headers,
       body: req.body,
-      dependencies: stateConfig.asyncTokenDependencies,
+      dependencies: asyncTokenConfig.dependencies,
+    });
+
+    res.status(result.statusCode);
+    res.send(JSON.parse(result.body));
+  });
+
+  app.post("/async/credential", async (req: Request, res: Response) => {
+    const result = await asyncCredentialHandlerConstructor({
```

Comment:
Do we feel there's value in abstracting the `asyncCredentialHandlerConstructor`? It feels a bit like it's creating extra code

Reply:
removed when pairing